Dockerfile

```diff
@@ -0,0 +1,35 @@
+# python 3
+FROM continuumio/anaconda3
+
+MAINTAINER Attawit Kittikrairit
+
+# set working directory
+WORKDIR /usr/src/app
+
```

Comment:
Don't you need to install mongodb as well?

Reply:
I will be submitting Dockerfile and instruction for mongodb separately on `e-mission-docs` repo.

Reply:
Or should we include mongodb script in this repo??

Reply:
I think that the `mongodb` script should go into this repo. Think about it:
- is mongodb needed if you are running the server? Yes
- is mongodb not needed if you are not running the server? Yes

So basically, mongodb is a server dependency and only a server dependency. It should go here.

Reply:
Check out my last commit